tor/net.go

```diff
@@ -71,13 +84,15 @@ type ProxyNet struct {
 
 // Dial uses the Tor Dial function in order to establish connections through
 // Tor. Since Tor only supports TCP connections, only TCP networks are allowed.
-func (p *ProxyNet) Dial(network, address string) (net.Conn, error) {
+func (p *ProxyNet) Dial(
```

Comment:
nit: line wrapping is a bit weird on these, network/address can be on the same line as `Dial`, then newline the rest

Reply:
fixed.